### PR TITLE
New blrasdfsd post

### PR DESCRIPTION
**Pull request by bcardarella**

### CODE REVIEW

File: source/posts/2015-04-09-the-beginner-fallacy.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+is especially evident when they are taken out of their current
+development environment.
+
+Beginners require guidance and a set of rules to follow. This has been
+shown time over time to be an effective strategy to onboarding. The
+rules/principles/laws, whatever you want to call them, are based upon
+the experience of more advanced developers that essentially boil down to 
+"here is what you want to do *most* of the time". They are a great
+substitute for experience. They also fit in very nicely with the notion
+of **Convetion Over Configuration**.
```
Comment (duggiefresh) [inline source/posts/2015-04-09-the-beginner-fallacy.md]: **convention**

File: source/posts/2015-04-09-the-beginner-fallacy.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+I am a big fan of Convention Over Configuration. I was first exposed to
+it with Ruby on Rails and I am a big fan that Ember.js has been
+following this path as well. However, I also believe that it can become
+a cruth for beginners. The rules that they are following coupled with
```
Comment (duggiefresh) [inline source/posts/2015-04-09-the-beginner-fallacy.md]: crutch

File: source/posts/2015-04-09-the-beginner-fallacy.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+the low friction environment of COC leads to a developer experience that
+does not present too many obstacles to be overcome. Learning requires
+challenge. You meet a challenge, you learn how to overcome that
+challege, you move on and now that experience is a tool you can wield in
```
Comment (duggiefresh) [inline source/posts/2015-04-09-the-beginner-fallacy.md]: challenge

File: source/posts/2015-04-09-the-beginner-fallacy.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+favored towards the employer. In most cases these developers will be
+able to accomplish most of what a very experienced developer can
+accomplish. This is great for the company's bottom line but not so good
+for the individual developer's own career growth.
+
+Instead, if you are interested in how do you advance beyond a beginner
+you must get out of your comfort zone. This is going to require you to
+put more time into your craft than you job necessarily allows. Try new
+software languages, try old software languages. Go read
+[SICP](https://mitpress.mit.edu/sicp/). Attend meetups, read papers,
```
Comment (duggiefresh) [inline source/posts/2015-04-09-the-beginner-fallacy.md]: There's `papers-we-love`, a GitHub repo with a lot of CS papers. https://github.com/papers-we-love/papers-we-love

Maybe we can link out to it for `read papers` portion?

File: source/posts/2015-04-09-the-beginner-fallacy.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "The Beginner Fallacy"
```
Comment (poteto) [inline source/posts/2015-04-09-the-beginner-fallacy.md]: `Beginner's`

File: source/posts/2015-04-09-the-beginner-fallacy.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+the future.
+
+The market being flooded with beginners along with COC frameworks should
+produce an employment environment for product companies that is heavily
+favored towards the employer. In most cases these developers will be
+able to accomplish most of what a very experienced developer can
+accomplish. This is great for the company's bottom line but not so good
+for the individual developer's own career growth.
+
+Instead, if you are interested in how do you advance beyond a beginner
```
Comment (poteto) [inline source/posts/2015-04-09-the-beginner-fallacy.md]: How about:

```
Instead, if you're interested in advancing beyond being a beginner, 
you must get out of your comfort zone.
```

File: source/posts/2015-04-09-the-beginner-fallacy.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+you must get out of your comfort zone. This is going to require you to
+put more time into your craft than you job necessarily allows. Try new
```
Comment (poteto) [inline source/posts/2015-04-09-the-beginner-fallacy.md]: `your`

**Comment by poteto** [comment]
LGTM

**Pull request merged**